alvr/client_openxr/src/lib.rs

```diff
@@ -120,7 +120,7 @@ pub fn entry_point() {
 
     let loader_suffix = match platform {
         Platform::Quest1 => "_quest1",
-        Platform::PicoNeo3 => "_pn3",
+        Platform::PicoNeo3 | Platform::PicoG3 | Platform::Pico4 => "_pico_old",
```

Comment:
Didn't we just change this in a pr? Also pico4 with the pico specific loader seems odd

Reply:
I found out that the Pico 4 Enterprise (the one supporting eye tracking) is still on an old firmware requiring the old loader. I don't have enough info to know which `DEVICE` string that corresponds to